### PR TITLE
fix: handle duckduckgo no result

### PR DESCRIPTION
**Pull request by elianiva**
finna go to bed

### CODE REVIEW

File: src/services/laodeai/index.js [hunk 1]
```diff
@@ -81,19 +81,19 @@ async function laodeai(context) {
...
-  const validSources = $('.web-result')
-    .map((_, el) => {
+  const sources = $('.web-result').get();
+  if (sources.length <= 1 && $(sources[0]).find('.no-results').get()) {
+    await context.reply("Uhh, I don't have an answer for that, sorry.");
+    return;
+  }
+
+  const validSources = sources
+    .map((el) => {
```
Comment (aldy505) [inline src/services/laodeai/index.js:91]: Shouldn't we still need to handle if they're still no valid sources available before line 97?

**Review by aldy505 — CHANGES_REQUESTED** [review]

File: src/services/laodeai/index.js (hunk 1 above)
Comment (elianiva) [reply src/services/laodeai/index.js:91]: sek, maksudnya gimana, aku gagal paham

Comment (aldy505) [reply src/services/laodeai/index.js:91]: itu kan di line atasnya kamu cuma handle kalo DDG emang ga punya result.

tapi dibawahnya kan nyari validSources sebelom di scrape lagi links nya sama DDG. what if there are no available valid sources?

Comment (elianiva) [reply src/services/laodeai/index.js:91]: oh iya lupa, ok

**Review by aldy505 — APPROVED** [review]

**Pull request merged**